src/jdk.httpserver/share/classes/sun/net/httpserver/Event.java

```diff
@@ -25,14 +25,33 @@
 
 package sun.net.httpserver;
 
-import com.sun.net.httpserver.*;
-import com.sun.net.httpserver.spi.*;
-
 class Event {
 
     ExchangeImpl exchange;
 
     protected Event (ExchangeImpl t) {
         this.exchange = t;
     }
+
```

Comment:
I don't see the purpose of this change and the value it brings other than collation and reducing explicit files for the derived Events ?
Could you elaborate the rationale please?

I do see a code smell in WriteFinished directly access the state of ExchangeImpl